Documentation/spec/docfx_document_schema.md

```diff
@@ -255,3 +255,13 @@ Here's the schema to describe these operations:
             2. Little chance that keywords DocFX defines duplicate with what JSON schema defines, after all, JSON schema defines a finite set of reserved keywords.
             3. For example[Swagger spec](http://swagger.io/) is also based on JSON schema and the fields it introduces in has no prefix. 
     * Decision: *Remove* `d-` prefix.
+3. What's remaining work if to apply schema to the complex data model, for example, ManagedReference, or UniversalReference?
+    * 1. OPS plugin framework, to insert metadata into the data model, for example, git commit id, git contributers.
+      Solution: A TagInterpreter plugin framework to insert metadata if the property contains `metadata` tag
```

Comment:
The same tag may have different behavior in different schema. Is this supported? e.g. gitcomit's definition is different between conceptual and mref